assets/stylesheets/site/components/_custom.scss

```diff
@@ -83,96 +97,109 @@ button.usa-menu-btn,
 }
 
 .main-content {
-  margin-bottom: 6rem;
-  margin-top: 1.5rem;
+  margin-bottom: units(8);
+  margin-top: units(2);
 
-  @include media($nav-width) {
-    margin-top: 5rem;
+  @include at-media($nav-width) {
+    margin-top: units(6);
   }
 }
 
-@include media($nav-width) {
-  .usa-header-basic {
+@include at-media($nav-width) {
+  .usa-header--basic {
     .usa-navbar {
       width: 350px;
     }
 
     .usa-logo {
-      bottom: 1.9rem;
+      bottom: units(2.5);
     }
   }
 
   .page-home {
-    .usa-header-basic {
+    .usa-header--basic {
       .usa-navbar {
         width: 100%;
       }
     }
   }
 }
 
-@include media($nav-width) {
-  .usa-navbar {
-    height: 7.3rem;
-  }
-}
+
 
 // [role="navigation"] {
-//   @include media($nav-width) {
+//   @include at-media($nav-width) {
 //     background-color: transparent;
 //   }
 // }
 
 .usa-nav {
-  padding-top: 1.6rem;
+  background: white;
+  padding-top: units(2);
 }
 
-.usa-nav-primary > li > a {
-  @include media($nav-width) {
+.usa-nav__primary > li > a {
+  @include at-media($nav-width) {
     // color: #cad9ef;
+    padding-bottom: 22px !important;
   }
 
   &.usa-page-current {
-    @include media($nav-width) {
+    @include at-media($nav-width) {
       // color: $color-white;
     }
 
     > span {
-      @include media($nav-width) {
-        border-bottom: 0.7rem solid darkslateblue;
+      @include at-media($nav-width) {
+        border-bottom: 7px solid darkslateblue;
         color: $color-base;
-        padding-bottom: 1rem;
```

Comment:
as a note, the base font size is no longer 10x, so the easy `rem` scale of .1 no longer applies. There's a theme customization for changing what the `rem` base unit is, but I found it buggy. This could have been addressed, but using the functions and tokens does a better job keeping the scales in check.